W 11239241 internet browser change cp

Crispy-Salesforce opened the pull request:
W-11239241 Changing some explanation regarding the default Internet browser.

Crispy-Salesforce committed:
Adding missing features in Windows for Studio 6.x

Crispy-Salesforce committed:
Small fix

mdaloia commented on modules/ROOT/pages/index.adoc:
```diff
@@ -11,8 +11,12 @@ The same editor also allows you to edit RAML and WSDL API specification files an
 
-* Internet Explorer versions 11 and earlier are not supported. +
-Anypoint Studio uses your system's native browser to display elements of the Anypoint Platform UI. If your system defaults to an Internet Explorer browser, you won't be able to visualize any of the Anypoint Platform views.
+* Studio 6.x will miss the following features in Windows:
+** Internet Explorer version 11 and earlier are not supported.
+** Anypoint Studio will not display the Anypoint Platform views if your system has a default Internet browser.
```
I would change this by something like:

> ... features in Windows due to Internet Explorer is not supported anymore by Anypoint Platform (See https://docs.mulesoft.com/general/browser-support):

and remove these last 2 sub-items because:
- Not a single one IE version is supported anymore
- it gives the sensation that there is a workaround configuring other browser by default, which is not, it depends on the native browser of Windows. 

Maybe it could be added something about that some Web UI parts could be still working but in the future could not. This will depend on the development of those parts of the web services on the Anypoint Platform.

Crispy-Salesforce replied:
I've made some changes, but I'm not sure I understood your comments. Please, can you take a look at it again? Thanks!

Crispy-Salesforce replied:
Thanks for the comments!
I've made some changes based on your comments.
If some parts of the web UI will not work in the future, it does not seem efficient to comment about it now.
Please, can you take a look at them and let me know if they are OK? Thanks!

mdaloia commented on modules/ROOT/pages/index.adoc:
```diff
@@ -11,8 +11,12 @@ The same editor also allows you to edit RAML and WSDL API specification files an
 
-* Internet Explorer versions 11 and earlier are not supported. +
-Anypoint Studio uses your system's native browser to display elements of the Anypoint Platform UI. If your system defaults to an Internet Explorer browser, you won't be able to visualize any of the Anypoint Platform views.
+* Studio 6.x will miss the following features in Windows:
+** Internet Explorer version 11 and earlier are not supported.
+** Anypoint Studio will not display the Anypoint Platform views if your system has a default Internet browser.
```
I would change this by something like:

> ... features in Windows due to Internet Explorer is not supported anymore by Anypoint Platform (See https://docs.mulesoft.com/general/browser-support):

and remove these last 2 sub-items because:
- Not a single one IE version is supported anymore
- it gives the sensation that there is a workaround configuring other browser by default, which is not, it depends on the native browser of Windows. 
- - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -

Crispy-Salesforce replied:
I've made some changes, but I'm not sure I understood your comments. Please, can you take a look at it again? Thanks!

Crispy-Salesforce replied:
Thanks for the comments!
I've made some changes based on your comments.
If some parts of the web UI will not work in the future, it does not seem efficient to comment about it now.
Please, can you take a look at them and let me know if they are OK? Thanks!

mdaloia commented on modules/ROOT/pages/index.adoc:
```diff
@@ -21,6 +25,7 @@ Anypoint Studio uses your system's native browser to display elements of the Any
 ** Other supported runtime versions work fine with both Java 7 or 8.
 
 * Studio 6 is not compatible with Mule 4.
+* From Studio 6 on, in Windows, you cannot use Exchange UI or deploy Mule apps to CloudHub.
```
I would not duplicate the info here... if you want to add something maybe I would add a reference to the previous paragraph.

mdaloia replied:
also, it really isn't from Studio 6 that it is not supported... it is since IE is not supported anymore on the Platform as previously commented... this doesn't mean that from one day to another all the platform doesn't work... for example there are parts that still work but some services like Exchange doesn't allow anymore to access with IE.

Crispy-Salesforce replied:
Good catch on your first comment!
Regarding IE, tough some parts may work, if we are stating that IE is not compatible with our platform, we are not responsible for the outcome of using our platform with IE, i.e.: it may work, it may not work, or it may render an unexpected result.
Thanks!

mdaloia commented on modules/ROOT/pages/index.adoc:
```diff
@@ -21,6 +25,7 @@ Anypoint Studio uses your system's native browser to display elements of the Any
 ** Other supported runtime versions work fine with both Java 7 or 8.
 
 * Studio 6 is not compatible with Mule 4.
+* From Studio 6 on, in Windows, you cannot use Exchange UI or deploy Mule apps to CloudHub.
```
I would not duplicate the info here... if you want to add something maybe I would add a reference to the previous paragraph.

Crispy-Salesforce replied:
Good catch!
Thanks!

mdaloia reviewed (commented):
Is the branch `v6.5`?  I ask because the latest Studio 6 is 6.6.10

Crispy-Salesforce committed:
Update index.adoc

Crispy-Salesforce commented:
> Is the branch `v6.5`? I ask because the latest Studio 6 is 6.6.10

You're right that 6.6.10 is the latest Studio version, but it is documented in 6.5 branch ;)

Crispy-Salesforce committed:
Update index.adoc

Crispy-Salesforce committed:
Update index.adoc

@mdaloia, please, can you take a look at the article now? I've made some changes. Thanks!

mdaloia commented on modules/ROOT/pages/index.adoc:
```diff
@@ -11,8 +11,11 @@ The same editor also allows you to edit RAML and WSDL API specification files an
-* Internet Explorer versions 11 and earlier are not supported. +
-Anypoint Studio uses your system's native browser to display elements of the Anypoint Platform UI. If your system defaults to an Internet Explorer browser, you won't be able to visualize any of the Anypoint Platform views.
+* Studio 6.x will miss some features in Windows due to the fact that Anypoint Platform no longer supports Internet Explorer (see https://docs.mulesoft.com/general/browser-support):
+** Anypoint Studio will not display the Anypoint Platform views if your system has a default Internet browser.
```
this line must be removed... causes confusion... It gives false sense that there is a workaround and there isn't really.

One thing that maybe could be added is that the below bullets are the currently ones that does not work and maybe in the future there will be others when more services start to remove support to IE.

Crispy-Salesforce replied:
Thanks for the explanations! ;)

mdaloia commented on modules/ROOT/pages/index.adoc:
```diff
@@ -11,8 +11,11 @@ The same editor also allows you to edit RAML and WSDL API specification files an
 
-* Internet Explorer versions 11 and earlier are not supported. +
-Anypoint Studio uses your system's native browser to display elements of the Anypoint Platform UI. If your system defaults to an Internet Explorer browser, you won't be able to visualize any of the Anypoint Platform views.
+* Studio 6.x will miss some features in Windows due to the fact that Anypoint Platform no longer supports Internet Explorer (see https://docs.mulesoft.com/general/browser-support):
```
I think that the practice in docs is not to put the direct link but a text with an internal link that it is resolved when docs are built. If it is not possible for some reason then leave a linked text for that URL.

Crispy-Salesforce replied:
You're right!
Thanks for the reminder! ;)

Crispy-Salesforce committed:
Update index.adoc

hannanelson commented on modules/ROOT/pages/index.adoc:
```diff
@@ -11,8 +11,12 @@ The same editor also allows you to edit RAML and WSDL API specification files an
 
-* Internet Explorer versions 11 and earlier are not supported. +
-Anypoint Studio uses your system's native browser to display elements of the Anypoint Platform UI. If your system defaults to an Internet Explorer browser, you won't be able to visualize any of the Anypoint Platform views.
+* Studio 6.x will miss some features in Windows due to the fact that Anypoint Platform no longer supports Internet Explorer (see xref:docs-general::browser-support.adoc[Browser Support for Anypoint Platform]).
```
```suggestion
* Studio 6.x doesn't use some features in Windows because Anypoint Platform no longer supports Internet Explorer (see xref:docs-general::browser-support.adoc[Browser Support for Anypoint Platform]).
```

hannanelson commented on modules/ROOT/pages/index.adoc:
```diff
@@ -11,8 +11,12 @@ The same editor also allows you to edit RAML and WSDL API specification files an
-* Internet Explorer versions 11 and earlier are not supported. +
-Anypoint Studio uses your system's native browser to display elements of the Anypoint Platform UI. If your system defaults to an Internet Explorer browser, you won't be able to visualize any of the Anypoint Platform views.
+* Studio 6.x will miss some features in Windows due to the fact that Anypoint Platform no longer supports Internet Explorer (see xref:docs-general::browser-support.adoc[Browser Support for Anypoint Platform]).
+** Authentication over MFA.
```
```suggestion
** Authenticate with MFA.
```

hannanelson replied:
(To make it parallel with the other bullets.)

hannanelson commented on modules/ROOT/pages/index.adoc:
```diff
@@ -11,8 +11,12 @@ The same editor also allows you to edit RAML and WSDL API specification files an
+** Access Exchange web UI.
+** Deploy to CloudHub.
+
+As IE is no longer supported, other features may not work in the future.
```
```suggestion
Because Internet Explorer is no longer supported, other features might not work in the future.
```

hannanelson replied:
(may implies permission; use might for probability)

Crispy-Salesforce replied:
Like it! Thanks!

Crispy-Salesforce committed:
Update modules/ROOT/pages/index.adoc

Co-authored-by: Hanna Nelson <hanna.nelson@mulesoft.com>

Crispy-Salesforce committed:
Update modules/ROOT/pages/index.adoc

Co-authored-by: Hanna Nelson <hanna.nelson@mulesoft.com>

Crispy-Salesforce committed:
Update modules/ROOT/pages/index.adoc

Co-authored-by: Hanna Nelson <hanna.nelson@mulesoft.com>
